website/src/pages/product/code-change-management.tsx

```diff
@@ -92,7 +94,14 @@ export default ((props: any) => (
                             }
                         />
                     </div>
-                    <div className="col-md-6">
+
+                    <div className="col-md-4">
+                        <Blockquote
+                            quote="... rules mitigating churn: infrastructure teams must do do the work to move their internal users to new versions themselves or do the update in place, in backwards-compatible fashion. This policy, which we've called the "Churn Rule," scales better: dependent projects are no longer spending progressively greater effort to just keep up."
```

Comment:
The quote looks good, but this syntax is incorrect. @aileenrose can you help?

Reply:
fixed quote and recognizing O'Reilly book